esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -128,8 +131,8 @@ void setup()
 
     size_t rxbufsize = SERIAL.setRxBufferSize(2*1024); // must come before uart started, retuns 0 if it fails
     size_t txbufsize = SERIAL.setTxBufferSize(512); // must come before uart started, retuns 0 if it fails
-#ifdef SERIAL_RXD // if SERIAL_TXD is not defined the compiler will complain, so all good
-    SERIAL.begin(baudrate, SERIAL_8N1, SERIAL_RXD, SERIAL_TXD);
+#ifdef SERIAL_RXD // if SERIAL_TXD or SERIAL_INVERT is not defined the compiler will complain, so all good
```

Comment:
my comment on this seems to have disappeared, sory for that
could you please change it as follows so that one can define rxd,txd without invert

#ifdef SERIAL_RXD
#ifdef SERIAL_INVERT
Serial.begin(..., SERIAL_INVERT);
#else
Serial.begin(...);
#endif
#endif

Reply:
This makes little sense to me, especially since you didn't allow defining SERIAL_RXD without SERIAL_TXD.  But, it's not worth debate.

Reply:
LOL